readme.md

```diff
@@ -110,6 +110,7 @@
 - [Web Accessibility](https://github.com/brunopulis/awesome-a11y)
 - [Material Design](https://github.com/sachin1092/awesome-material)
 - [CSS](https://github.com/sota0805/awesome-css)
+- [Gulp](https://github.com/alferov/awesome-gulp)
```

Comment:
Hmmm
@sindresorhus, @arthurvr, what do you think about a new category with the following structure:
#### Automated Development Workflow (or Workflow Automation)
- Gulp

In that case, it could be extended with Grunt and Broccoli later if someone decides to create and support these lists.